python/tk_multi_loader/dialog.py

```diff
@@ -1418,9 +1418,20 @@ def _setup_hierarchy_model(self, app, root):
         :return: Created `(model, proxy model)`.
         """
 
+        # If the root is a aproject, include it in the hierarchy model so that
+        # we can display project publishes. We do an innocent little hack here
+        # by including a space at the front of the project root item to make it
+        # display first in the tree.
+        include_root = " Project Publishes" if root.get("type") == "Project" else None
```

Comment:
I know its more complicated, but I think it would be great if someone on the team got to learn how this works so we can do cooler things with proxy models in the future.

Also, maybe its because I know, but the spacing between the icon and the text shows. :p